src/NuGet.Clients/NuGet.Tools/NuGet.Tools.csproj

```diff
@@ -68,98 +60,26 @@
-    </Content>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="..\VsConsole\PowerShellHost\Scripts\about_NuGet.PackageManagement.PowerShellCmdlets.help.txt">
-      <Link>Modules\NuGet\about_NuGet.PackageManagement.PowerShellCmdlets.help.txt</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="..\VsConsole\PowerShellHost\Scripts\NuGet.PackageManagement.PowerShellCmdlets.dll-Help.xml">
-      <Link>Modules\NuGet\NuGet.PackageManagement.PowerShellCmdlets.dll-Help.xml</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="$(SolutionPackagesFolder)Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll">
-      <Link>Newtonsoft.Json.dll</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="$(SolutionPackagesFolder)Microsoft.Web.Xdt.2.1.1\lib\net40\Microsoft.Web.XmlTransform.dll">
-      <Link>Microsoft.Web.XmlTransform.dll</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="$(SolutionPackagesFolder)Lucene.Net.3.0.3\lib\NET40\Lucene.Net.dll">
-      <Link>Lucene.Net.dll</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Resources\nuget_256.png">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Resources\nuget_96.png">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <VSCTCompile Include="NuGetTools.vsct">
-      <ResourceName>Menus.ctmenu</ResourceName>
-      <SubType>Designer</SubType>
-    </VSCTCompile>
-    <Content Include="..\VsConsole\PowerShellHost\Scripts\Add-WrapperMembers.ps1">
-      <Link>Modules\NuGet\Add-WrapperMembers.ps1</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="..\VsConsole\PowerShellHost\Scripts\NuGet.Format.ps1xml">
-      <Link>Modules\NuGet\NuGet.Format.ps1xml</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="..\VsConsole\PowerShellHost\Scripts\NuGet.psd1">
-      <Link>Modules\NuGet\NuGet.psd1</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="..\VsConsole\PowerShellHost\Scripts\nuget.psm1">
-      <Link>Modules\NuGet\nuget.psm1</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="..\VsConsole\PowerShellHost\Scripts\NuGet.Types.ps1xml">
-      <Link>Modules\NuGet\NuGet.Types.ps1xml</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="..\VsConsole\PowerShellHost\Scripts\Profile.ps1">
-      <Link>Modules\NuGet\Profile.ps1</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
     <None Include="project.json" />
-    <None Include="Resources\nugetAbout.ico" />
-    <None Include="Resources\Toolbar.png" />
   </ItemGroup>
-  <ItemGroup>
+   <ItemGroup>
```

Comment:
extra space. Please use template enforcing IDE/platform. So that you wont have to do these manually.